app/module-name.js

```diff
@@ -0,0 +1,19 @@
+const _s = require('underscore.string');
+
+const scopedRegex = /^(?:@([^/]+?)[/])?([^/]+?)$/;
+
+exports.isScoped = name => {
+	return scopedRegex.test(name);
+};
```

Comment:
This could be a one-liner:

```js
exports.isScoped = name => scopedRegex.test(name);
```

Same with the below. Just use a ternary instead.